feature-detects/es6/templatestrings.js

```diff
@@ -15,7 +15,7 @@
 Check if browser implements ECMAScript 6 String template.
 */
 define(['Modernizr'], function(Modernizr) {
-  Modernizr.addTest('stringtemplate', function() {
+  Modernizr.addTest('templatestrings', function() {
```

Comment:
why the rename? its just breaks both tests this way

Reply:
doing it for consistency, and this major version is the best time to break stuff

Reply:
sure, but to what end? why do it at all?

Reply:
because its an es6 feature and the spec is also called "template-strings" 
people would look for that and not the other feature name

Reply:
both bring up the same search, I really don't think we shouold rename it but if you strongly think we then could you add an `alias`  to the metadata

Reply:
an alias sounds reasonable, will add one

Reply:
done